utilities.py

```diff
@@ -0,0 +1,37 @@
+
```

Comment:
I like the idea of a separate file for utility functions, but since you're not using it, take it out of git.